### PR TITLE
Old High German

### PR DESCRIPTION
**Pull request by Viridianus**
First commit is for councilors, I will slowly go on with this.

### CODE REVIEW

**Comment by Viridianus** [comment]
Nevermind that it wants to be added to Master-Fixes; if they get accepted first, I'll just change the branch to Master (same for #83). I think I did it, updating quite a lot of titles along the way thanks to @Urkaun and Übergold.

File: common/customizable_localization/00_ctp_councillor_custom_loc.txt [hunk 1]
```diff
@@ -226,7 +226,7 @@ CouncilPositionChancellor = {
 				liege.culture = culture:swabian
 			}
 		}
-		localization_key = councillor_chancellor_bavarian_franconian_swabian
+		localization_key = councillor_chancellor_old_high_german ## Franconians, Swabians and Bavarians all spoke Old High German (c) Urkaun
```
Comment (Gr770) [inline common/customizable_localization/00_ctp_councillor_custom_loc.txt:229]: 👍

**Review by Gr770 — COMMENTED** [review]
Are independent counts not also 'princes?'

**Comment by Viridianus** [comment]
> Are independent counts not also 'princes?'
I don't think so, looks more like an artifact. Though it's a question for @Urkaun in the end.

**Pull request merged**